kubernetes/base/config/__init__.py

```diff
@@ -33,7 +33,10 @@ def load_config(**kwargs):
     can be passed to either load_kube_config or
     load_incluster_config functions.
     """
-    if "kube_config_path" in kwargs.keys() or exists(expanduser(KUBE_CONFIG_DEFAULT_LOCATION)):
+    if "kube_config_path" in kwargs.keys():
```

Comment:
it seems `kube_config_path` never worked. I wonder if we have any user setting it

Reply:
I think it might be due to the `load_kube_config` API change, `kube_config_path` is a ghost value now.